weaviate-services/agents/transformation-agent-get-started.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p><code>papers</code> -&gt; <code>transformation-agent-papers</code> ?</p><p><code>collections</code> -&gt; <code>a collection</code> ?</p><p> </p><p>I think this <code>We recommend enableing 'Embeddings' in the Weavaite Cloud console.&nbsp;</code> might need to be stronger, as this step will fail if it is not done first. </p><p> </p><p>Wdyt about: <code>IMPORTANT: Before running this code, enable ...</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/weaviate/recipes/pull/209/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>